addons/event_sale/tests/test_event_sale.py

```diff
@@ -279,3 +280,22 @@ def test_cancel_so(self):
         self.assertEqual(event.seats_expected, 1)
         self.sale_order.action_cancel()
         self.assertEqual(event.seats_expected, 0)
+
+    def test_check_event_id_and_event_ticket_id(self):
```

Comment:
A short docstring would be welcome too. Like """Ensure events created from sale orders lines have an associated event."""